### PR TITLE
{WIP - do not merge!} DIT-394 & DIT-396: search by commodity code and/or good description

### PR DESCRIPTION
**Pull request by gokyo**

### CODE REVIEW

File: test/it/uk/gov/hmrc/component/CaseSpec.scala [hunk 1]
```diff
@@ -482,4 +433,78 @@ class CaseSpec extends BaseFeatureSpec {
 
   }
 
+
+  feature("Get Cases by commodity code") {
+    // TODO
```
Comment (ed0906) [inline test/it/uk/gov/hmrc/component/CaseSpec.scala]: Presume these will be done as part of this PR?

File: test/it/uk/gov/hmrc/component/CaseSpec.scala [hunk 2]
```diff
@@ -482,4 +433,78 @@ class CaseSpec extends BaseFeatureSpec {
...
+  }
+
+
+  feature("Get Cases sorted by commodity code") {
+    // TODO
+  }
+
+
+  feature("Get Cases sorted by all possible sort fields") {
+    // TODO: test both sort directions
```
Comment (ed0906) [inline test/it/uk/gov/hmrc/component/CaseSpec.scala]: Not a requirement

File: test/it/uk/gov/hmrc/component/CaseSpec.scala (hunk 1 above)
Comment (gokyo) [reply test/it/uk/gov/hmrc/component/CaseSpec.scala]: yes. I did not have time yesterday

File: test/it/uk/gov/hmrc/component/CaseSpec.scala (hunk 2 above)
Comment (gokyo) [reply test/it/uk/gov/hmrc/component/CaseSpec.scala]: ok

**Comment by gokyo** [comment]
Closing this old PR

**Pull request closed without merging**